maps/_map.dm

```diff
@@ -146,6 +146,18 @@ proc/get_base_turf(var/z)
 	var/datum/zLevel/L = map.zLevels[z]
 	return L.base_turf
 
+proc/change_base_turf(var/choice,var/new_base_path,var/update_old_base = 0)
+	if(update_old_base)
+		for(var/turf/T in turfs)

```

Comment:
I don't see why that would give those results but it doesn't seem to make a big difference either way compared to the rest of the proc it's being called in